fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -169,6 +170,15 @@ default FDBStoreTimer getTimer() {
     @Nullable
     SubspaceProvider getSubspaceProvider();
 
+    /**
+     * Get the {@link RecordStoreState} for this store.
+     * This represents the indexes that are disabled or in the process of being rebuilt.
+     * If the state is not already loaded, it is loaded synchronously.
+     * @return the store state for this store
+     */
+    @Nonnull
+    RecordStoreState getRecordStoreState();
```

Comment:
All changes to the record layer should have an accompanying issue, and updates to the release notes.